happy/HappyInternet.py

```diff
@@ -42,6 +42,7 @@
 options["isp"] = None
 options["seed"] = None
 
+IPTABLES_CONFIG = "/tmp/default.fw"
```

Comment:
I don't think this works with multiple instances of happy.  Can this be stored on a per-instance state of the happy state?